akka-sse/src/main/scala/de/heikoseeberger/akkasse/ServerSentEventParser.scala

```diff
@@ -31,31 +32,23 @@ private object ServerSentEventParser {
 
   private final val Retry = "retry"
 
-  private val linePattern = """([^:]+): ?(.*)""".r
+  private val LinePattern = """([^:]+): ?(.*)""".r
```

Comment:
I only use capitalized identifiers for compile time constants. Please lower case again.